recipe/build.sh

```diff
@@ -18,6 +18,7 @@ fi
     --localstatedir=${PREFIX}/var \
     --with-vpnc-script=${PREFIX}/etc/openconnect/vpnc-script \
     --disable-flask-tests \
+    --with-external-browser \
```

Comment:
`--with-external-browser` is not needed needed since `xdg-open` is autodetected. I would only specify this flag in situations where we want to explicitly disable the `libopenconnect` default callback or override with a custom browser choice.

Reply:
Hi Leo,

It's not unsafe as you said. It's more of a preference.